seqr/management/commands/lift_project_to_hg38.py

```diff
@@ -82,7 +82,7 @@ def handle(self, *args, **options):
                 ))
 
         # Lift-over saved variants
-        _update_variant_samples(matched_sample_id_to_sample_record, elasticsearch_index, dataset_path)
+        _update_variant_samples(matched_sample_id_to_sample_record, elasticsearch_index)
```

Comment:
By removing this you actually make this variable  unused complete, which gets flagged by our style checker. Could you remove the variable declaration above on line 42?

Reply:
Okay